PS8lite.Rmd

```diff
@@ -187,7 +187,25 @@ MiscVal + MoSold + YrSold" %>%
 Fit one Random Forest model `model_rf` using any approach you want.
 
 ```{r}
+# Fit random forest:
+model_rf <- randomForest(
+  # Model formula
+  form = model_formula,
+  # Training data
+  data = training, 
+  # At each node of tree, number of features/predictor variables to randomly
+  # choose from for splitting:
+  mtry = 2,
+  # Number of bagged (bootstrap aggregated) trees in your forest:
+  ntree = 100 
+)
 
+test <- test %>% 
+  mutate(
+    logSalePrice_hat_rf = predict(object = model_rf, newdata = test),
+    SalePrice_hat_rf = exp(logSalePrice_hat_rf) -1
+  )
+glimpse(test)
```

Comment:
Hi, everything looks good. Here is an example comment that I would leave as part of a code review if this were real life. Then you would then make the requested changes to this PR and then I would merge your changes into my master copy of the repo. I won't be doing that last step here.